nautobot/extras/tests/test_api.py

```diff
@@ -619,6 +628,140 @@ def test_delete_job_result_with_permission(self):
         self.assertHttpStatus(response, status.HTTP_204_NO_CONTENT)
 
 
+class ScheduledJobTest(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        user = User.objects.create(username="user1", is_active=True)
+        ScheduledJob.objects.create(
+            name="test1",
+            task="-",
+            job_class="-",
+            interval=JobExecutionType.TYPE_IMMEDIATELY,
+            user=user,
+            approval_required=True,
+            start_time=now(),
+        )
+        ScheduledJob.objects.create(
+            name="test2",
+            task="-",
+            job_class="-",
+            interval=JobExecutionType.TYPE_IMMEDIATELY,
+            user=user,
+            approval_required=True,
+            start_time=now(),
+        )
+        ScheduledJob.objects.create(
+            name="test3",
+            task="-",
+            job_class="-",
+            interval=JobExecutionType.TYPE_IMMEDIATELY,
+            user=user,
+            approval_required=True,
+            start_time=now(),
+        )
+
+    @override_settings(EXEMPT_VIEW_PERMISSIONS=["extras.scheduledjob"])
+    def test_list_objects(self):
+        count = ScheduledJob.objects.count()
+
+        response = self.client.get(reverse("extras-api:scheduledjob-list"), **self.header)
+        self.assertHttpStatus(response, status.HTTP_200_OK)
+        self.assertEqual(response.data["count"], count)
+
+    @override_settings(EXEMPT_VIEW_PERMISSIONS=["extras.scheduledjob"])
+    def test_get_object(self):
+        job = ScheduledJob.objects.first()
+
+        url = reverse("extras-api:scheduledjob-detail", kwargs={"pk": job.pk})
+        response = self.client.get(url, **self.header)
+        self.assertHttpStatus(response, status.HTTP_200_OK)
+        self.assertEquals(response.data["name"], job.name)
```

Comment:
Why are you overriding the base `APITestCase` `test_list_objects` and `test_get_object` test functions? Do they not work as written?

Reply:
Because I copied that behavior from another test case, unaware that `APITestCase` already comes with a valid implementation for these two functions. 🤦